x-pack/plugins/session_view/public/components/session_view_toggle_options/index.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiPopover,
+  EuiSelectable,
+  EuiPopoverTitle,
+  EuiFlexItem,
+  EuiButtonIcon,
+} from '@elastic/eui';
+
+interface OptionsField {
+  label: string;
+  value: string;
+  checked: 'on' | 'off' | undefined;
+}
+interface StateField {
```

Comment:
better to move this to a more common place to be shared with `session_view` stuff mentioned above